src/components/ShareModal.js

```diff
@@ -18,11 +18,7 @@ class ShareModal extends Component {
   state = {
     activeIndex: 0,
     emails: [],
-  };
-
-  copyUrlToClipboard = () => {
-    copyToClipboard(this.props.link);
-    // this.props.alert.success('URL Copied To Clipboard');
+    copied: false,
```

Comment:
What is this for? It doesn't look like it's used

Reply:
just in case we want some sort of alert in the future.
ok I'll remove this.

Reply:
I thought so, but then it'll be incomplete because this boolean is only toggled to true once, so repeated button clicks won't have any effect. Better to just implement this together with the notification so that the person doing that won't be constrained/confused by this partial implementation.